modules/cli-installing-cli.adoc

```diff
@@ -154,16 +139,16 @@ ifdef::openshift-origin[]
 . Navigate to link:https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/[https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/] and choose the folder for your operating system and architecture.
 . Download `oc.zip`.
 endif::[]
-ifndef::openshift-origin,microshift[]
+ifndef::openshift-origin,microshift,openshift-rosa,openshift-rosa-hcp,openshift-dedicated[]
 . Navigate to the link:https://access.redhat.com/downloads/content/290[{product-title} downloads page] on the Red Hat Customer Portal.
```

Comment:
as above you should never have the word `page` in the link and they have the incorrect page name.